From 9b4e3b9a661e6316b0fe2a966347fc8e9c39d552 Mon Sep 17 00:00:00 2001
From: Alexey Neyman <stilor@att.net>
Date: Wed, 26 Sep 2018 14:09:03 +0000
Subject: [PATCH] graphite.h: Include <isl/id.h> and <isl/space.h>...

	* graphite.h: Include <isl/id.h> and <isl/space.h>; these
	headers are no longer pulled in by <isl/val.h>.

From-SVN: r264642
---
 gcc/ChangeLog  | 5 +++++
 gcc/graphite.h | 2 ++
 2 files changed, 7 insertions(+)

diff --git a/gcc/graphite.h b/gcc/graphite.h
index be0a22b38942..8db5700e03f1 100644
--- a/gcc/graphite.h
+++ b/gcc/graphite.h
@@ -26,6 +26,8 @@ along with GCC; see the file COPYING3.  If not see
 #include <isl/options.h>
 #include <isl/ctx.h>
 #include <isl/val.h>
+#include <isl/id.h>
+#include <isl/space.h>
 #include <isl/set.h>
 #include <isl/union_set.h>
 #include <isl/map.h>
